Sources/SwiftDiagnostics/FixIt.swift

```diff
@@ -46,3 +46,44 @@ public struct FixIt {
     self.changes = changes
   }
 }
+
+public extension FixIt {
+  var edits: [SourceEdit] {
```

Comment:
Could you add a doc comment for `edits`?

--- 

Also a nitpick: I would prefer `extension` and `public var` over `public extension` and `var` because it makes it obvious that the variable is part of the public API. With `public extension` it’s quite easy to add a new member without a type annotation to the extension which then becomes part of the public API without notice.

Reply:
yeah, i'll add a doc comment and update the code due to above reason.